Automattic/wp-calypso
# Change promote post texts to match mockups

#### Proposed Changes

- Changed promote post educational banners and texts to match latest figma changes
- Mix posts and pages in /advertising endpoint

#### Testing Instructions

Go to wordpress.com/advertising, check that you can see both posts and pages in "Ready to Promote" list


#### Pre-merge Checklist

- [] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?

Related to #
